src/api/certifications/cerfitications.js

```diff
@@ -0,0 +1,111 @@
+// @flow
+type UpdateCertificationParams = {
+    description: string,
+    title: string,
+    type: string,
+    state: string
+}
+
+type UpdateCertificationsParams = {
+    certifications: [
+        {
+            description: string,
+            title: string,
+            type: string,
+            state: string
+        }
+    ]
+}
+
+type UpdateCustomerCertificationsParams = {
+    action: string,
+    certification_ids: []
+}
+
+type UpdateOrganizationCertificationsParams = {
+  certifications: [
+    {
+      id: number,
+      description: string,
+      title: string,
+      type: string,
+      state: string
+    }
+  ]
+}
+
+type DeleteOrganizationCertificationsParams = {
+    certification_ids: []
+}
+
+type CreateOrganizationCertificationsFromFileParams = {
+    s3_keys: [ ]
+}
+
+export default class Certificates {
+    constructor(client: Axios) {
+        this.client = client;
+    }
+
+    client: Axios;
+
+    deleteCertification(certificationID : number): Promise<AxiosXHR<Object>> {
```

Comment:
It might be good to describe what it actual return is: https://github.com/gigwalk-corp/gigwalk-node/blob/5ee9859b089415b0d228f48a976fdc42d88572c6/src/api/versions/index.js#L5